packages/shared/src/lib/history.ts

```diff
@@ -0,0 +1,134 @@
+import { LocalStorage, TagCounts, WeeklyArticleHistory, WeeklyArticleLog } from '../types/payloads';
+
```

Comment:
this file comes from automat - https://github.com/guardian/automat-client-v2/blob/master/src/contributions/history.ts